modules/userId/index.js

```diff
@@ -150,15 +152,15 @@ function setStoredValue(storage, value) {
     const valueStr = utils.isPlainObject(value) ? JSON.stringify(value) : value;
     const expiresStr = (new Date(Date.now() + (storage.expires * (60 * 60 * 24 * 1000)))).toUTCString();
     if (storage.type === COOKIE) {
-      utils.setCookie(storage.name, valueStr, expiresStr, 'Lax');
+      coreStorage.setCookie(storage.name, valueStr, expiresStr, 'Lax');
       if (typeof storage.refreshInSeconds === 'number') {
-        utils.setCookie(`${storage.name}_last`, new Date().toUTCString(), expiresStr);
+        coreStorage.setCookie(`${storage.name}_last`, new Date().toUTCString(), expiresStr);
       }
     } else if (storage.type === LOCAL_STORAGE) {
-      utils.setDataInLocalStorage(`${storage.name}_exp`, expiresStr);
-      utils.setDataInLocalStorage(storage.name, encodeURIComponent(valueStr));
+      localStorage.setItem(`${storage.name}_exp`, expiresStr);
+      localStorage.setItem(storage.name, encodeURIComponent(valueStr));
```

Comment:
are these `localStorage` references correct?   Shouldn't they go to the `coreStorage` functions like the updates for the cookies above?

Reply:
+1